crates/cli/src/opts/wallet/mod.rs

```diff
@@ -42,11 +42,7 @@ pub struct RawWallet {
     pub interactive: bool,
 
     /// Use the provided private key.
-    #[clap(
-        long,
-        value_name = "RAW_PRIVATE_KEY",
-        value_parser = foundry_common::clap_helpers::strip_0x_prefix
```

Comment:
we no longer need this because alloy handles this?

Reply:
yup